api-integration-tests.sh

```diff
@@ -5,3 +5,4 @@ python3 concent_api/api-integration-force-accept-or-reject-results-test.py $1 &&
 python3 concent_api/api-integration-force-get-task-result-test.py $1 &&
 python3 concent_api/api-integration-force-payment.py $1 &&
 python3 concent_api/api-integration-force-report-computed-task-test.py $1
+python3 concent_api/api-integration-additional-verification-test.py $1
```

Comment:
`&&` is missing in the line above (or remove all of them; anyway, let's be consitent:))

Reply:
Done